mypy/test/testfinegrained.py

```diff
@@ -86,6 +86,7 @@ def run_case(self, testcase: DataDrivenTestCase) -> None:
         step = 1
         sources = self.parse_sources(main_src, step, options)
         if step <= num_regular_incremental_steps:
+            build_options.incremental = step > 1
```

Comment:
What's the purpose of this change?

Reply:
It was so that we'd split into multiple files on the first pass, but I got rid of that behavior